Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:2.7
+
+COPY . /securitybot
+
+env PYTHONPATH $PYTHONPATH:/securitybot
+
+RUN apt-get update && \
+    apt-get install -y mysql-client && \
```

Comment:
Ah, I see. I think a docker-compose file with a MySQL linked container would be useful in this repository - makes it quite simple to get everything un and running locally. 

The kubernetes config files seem a little out of scope, and something that probably wouldn't be maintained here. Let's leave that in another repository :)